packages/button/src/css/index.js

```diff
@@ -44,39 +44,15 @@ export default {
     border: '1px solid transparent'
   },
   '.psds-button:focus:before': {
-    content: ' ',
-    position: 'absolute',
-    top: '-1px',
-    left: '-1px',
-    right: '-1px',
-    bottom: '-1px',
-    background: core.colors.black,
-    zIndex: '-1',
-    borderRadius: '2px'
-  },
-  [`.psds-button.psds-theme--${themeNames.light}:focus:before`]: {
-    top: '-2px',
-    left: '-2px',
-    right: '-2px',
-    bottom: '-2px',
-    background: core.colors.bone
-  },
-  '.psds-button:focus:after': {
     content: ' ',
     position: 'absolute',
     top: '-4px',
     left: '-4px',
     right: '-4px',
     bottom: '-4px',
-    zIndex: '-2',
-    borderRadius: '4px',
-    background: core.colors.blue
-  },
-  [`.psds-button.psds-theme--${themeNames.light}:focus:after`]: {
-    top: '-5px',
-    left: '-5px',
-    right: '-5px',
-    bottom: '-5px'
+    zIndex: '-1',
+    border: `3px solid ${core.colors.blue}`,
```

Comment:
I love the simplification of this.  I am concerned that we've diverged from the original requirements of the focus indicator.  I'm also concerned that I don't remember them 100%.  It was quite the process to get them exactly right.  Perhaps we need to review what exactly they are.  This would be good to do before I tackle #289, which is going to address focus ring visibility for the other form-related components.  I propose a 15 minute meeting to review today.  @dborodyansky would you be able to walk us through the viz specs of the focus indicator today?

Reply:
totally agree. a quick design meeting would be helpful.

i could tell there was a clear reasoning for the complexity :)